src/kernel-install/60-ukify.install.in

```diff
@@ -187,7 +187,7 @@ def call_ukify(opts):
     # so it doesn't make sense to duplicate all the fields. We use a hack
     # to pre-populate the namespace like argparse would, all defaults.
     # We need to specify the two mandatory arguments to not get an error.
-    opts2 = ukify['create_parser']().parse_args(('A','B'))
+    opts2 = ukify['create_parser']().parse_args(('create','A','B'))
```

Comment:
also "create" → "make"? i.e. we just combine stuff here, we don't really create anything new. hence "build" or "make" sounds like better wording than "create" to me